curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md

```diff
@@ -9,6 +9,8 @@ dashedName: step-136
 
 In the `locations` array, add another object at the end. Set the `name` property to `"lose"`, set `"button text"` to an array with three `"REPLAY?"` strings, set `"button functions"` to an array with three `restart` variables, and set `text` to `"You die. &#x2620;"`. 
 
+In a later step, you will update a function for the &#x2620; emoticon text to properly display on the page.
```

Comment:
```suggestion
In a later step, you will fix the code in the `update` function to make the `&#x2620;` emoticon text properly display on the page.
```

How about this?

I think we at least need to add a code block for the emoticon text value.

Reply:
Happy to add back ticks for the emoticon code.
Will leave wording of new text for fCC staff to confirm.